src/class-parsely.php

```diff
@@ -812,4 +814,35 @@ private function sanitize_managed_option( string $option_id, $value ) {
 
 		return $value;
 	}
+
+	/**
+	 * Allows remote requests to Parse.ly.
+	 *
+	 * This is needed for environments, such as wp-now, that block remote requests.
+	 *
+	 * @since 3.13.0
+	 * @access private
+	 */
+	private function allow_parsely_remote_requests(): void {
+		$allowed_urls = array(
+			self::DASHBOARD_BASE_URL,
+			self::PUBLIC_API_BASE_URL,
+			self::PUBLIC_SUGGESTIONS_API_BASE_URL,
+		);
+
+		add_filter(
+			'http_request_host_is_external',
+			function ( $external, $host, $url ) use ( $allowed_urls ) {
+				// Check if the URL matches any URLs on the allowed list.
+				foreach ( $allowed_urls as $allowed_url ) {
+					if ( str_starts_with( $url, $allowed_url ) ) {
```

Comment:
`str_starts_with` is a PHP 8 function but we need to support PHP 7.2 and up.

Maybe it would be worth creating a utility function for it in `utils.php` and adding a comment that it can be replaced with `str_starts_with` when the minimum PHP support version becomes 8.

https://github.com/Automattic/vip-go-mu-plugins-built/blob/43ad825eb2dc3c6a8cd9f12144f162aca2ab6090/vip-helpers/vip-utils.php#L357-L359

Reply:
Ahh good catch, I completely forgot that caveat with the `str_starts_with`. Will add a fallback in `utils.php` :)